samples/openapi3/client/petstore/spring-cloud-oas3-fakeapi/src/main/java/org/openapitools/api/AnotherFakeApi.java

```diff
@@ -31,7 +31,7 @@
 
 @Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
 @Validated
-@Tag(name = "AnotherFake", description = "the AnotherFake API")
+@Tag(name = "$another-fake?", description = "the $another-fake? API")
```

Comment:
This is valid tag? if yes then probably some kind of escape required here?

Reply:
That exact same tag already exists in the file; this change merely aligns this tag to have the same value as the other tag.

https://github.com/OpenAPITools/openapi-generator/pull/13434/files/3d7c2cea196608e48cea4f9ce94230a649658987#diff-b54319fe49c2dd8b26f8e1e914fd1d81b5a516b44ae587a9e13a5cfb526cec80L47

Reply:
(bah - link didn't work. But it's on Line 47 of that file - unchanged in this PR)